src/bag3_testbenches/measurement/mos/sp.py

```diff
@@ -76,29 +90,51 @@ async def async_meas_pvt(self, name: str, sim_dir: Path, sim_db: SimulationDB, d
                           dict(pin='dbias', nin='d', type='dcfeed', value=''),
                           ])
 
-        # set vgs and vds sweep info
-        vds_val = self.specs['vds_val']
-        vgs_val = self.specs['vgs_val']
+        is_mc = 'monte_carlo_params' in self.specs['tbm_specs']
+        is_nmos: bool = self.specs['is_nmos']
 
         tbm_specs = dict(
             **self.specs['tbm_specs'],
             load_list=load_list,
             sim_envs=[pvt],
-            swp_info=[('vds', dict(type='LIST', values=vds_val)),
-                      ('vgs', dict(type='LIST', values=vgs_val))],
             dut_pins=dut.pin_names,
             param_type='Y',
             ports=['PORTG', 'PORTD'],
         )
+
+        if not is_mc:
+            vgs_sweep = self.specs['vgs_sweep']
```

Comment:
New parameters. Can we get a docstring with all the expected contents of self.specs?

Reply:
Sure, I will add docstrings.